tools/abacas/.shed.yml

```diff
@@ -0,0 +1,7 @@
+categories: [Contiguate]
```

Comment:
Change to **Assembly**, otherwise it will fail auto deployment since **Contiguate** does not exist on public toolshed.